test/probability_simplex.jl

```diff
@@ -54,13 +54,5 @@ include("utils.jl")
         X2 = X .+ 10
         Y = project(M, p, X2)
         @test isapprox(M, p, X, Y)
-
-        X = log(M, q, p)
```

Comment:
Why did you remove these tests?

Reply:
I hope not, but I had a strange merge, which I hope I aborted and restarted :/

Reply:
It's restored.